docker/iml-mailbox/iml-mailbox.service

```diff
@@ -0,0 +1,17 @@
+[Unit]
+Description=IML Mailbox Service
+After=syslog.target network.target
+
+
+[Service]
+Type=simple
+Environment=RUST_LOG=info
+EnvironmentFile=/var/lib/chroma/iml-settings.conf
+EnvironmentFile=/etc/systemd/system/iml-mailbox.conf
```

Comment:
Hrrm, I don't think you can have multiple `EnvironmentFile` directives.

Reply:
nm, you can.

Reply:
Apparently [it is ok](https://www.freedesktop.org/software/systemd/man/systemd.exec.html#EnvironmentFile=):

> This option may be specified more than once in which case all specified files are read. If the empty string is assigned to this option, the list of file to read is reset, all prior assignments have no effect.

Reply:
Yup, this threw me off at first. So the problem here is that we load the iml-settings file to load the environment variables. But we need to override the PROXY_HOST variable. My first thought was to just set an environment variable here in the service file thinking that it would override the value that was loaded from iml-settings.conf. But this doesn't work. After doing some research, I found an open issue for systemd in which others were trying to do the same thing and couldn't override the environment variables loaded from the EnvironmentFile directive. It turns out, systemd only lets you override a variable that's loaded from an environment file by loading an additional environment file with a variable containing the same name. Seems like a poor design in my opinion, but that appears to be the way to do it. It's easy to verify that this works because when the service loads it will indicate the IP and port in which the service is running. It will display the IP for the iml-timer domain but if we removed the EnvironmentFile for iml-mailbox.conf and replaced it with a standard environment variable you would see 127.0.0.1.